core/scripts/cli/sponsor/create.js

```diff
@@ -24,7 +24,7 @@ const create = async (web3, artifacts, emp) => {
   });
   // Use BigNumber.js to so that we can set ROUNDING_MODE to round ceiling. We need to do this to make sure we send
   // enough collateral to create the requested tokens.
-  BigNumber.set({ ROUNDING_MODE: 2 });
+  BigNumber.set({ ROUNDING_MODE: 2, RANGE: 500, EXPONENTIAL_AT: 500 });
```

Comment:
nit: can you document why you chose to set `RANGE` and `EXPONENTIAL_AT` to these values?

Reply:
Yup, good point.

Reply:
Done.